File: grader.py
import sys

name = raw_input("Enter student name: ")
assignment = raw_input("Enter assignment name: ")
grade = raw_input("Enter grade in numbers: ")

class Grader:
	def _init_(self,name,assignment,grade):
		self.name = name
		self.assignment = assignment
		self.grade = grade

	def getGrade(grade):
		if grade <= 100 and grade >= 90:
			grade == 'A'
	
		elif grade <= 89 and grade >= 80:
			grade == 'B'
		
		elif grade <= 79 and grade >= 70:
			grade == 'C'
		
		elif grade <= 69 and grade >= 60:
			grade == 'D'
		
		elif grade <60:
			grade == 'F'
		
		else:
			"Please enter a valid number for grade"
		
		return grade
	def displayGrader():
		print name+"@fullsail.edu\n"
	"Your grade for "+assignment+" is: "+grade



grader = Grader(name,assignment,grade)
grader.getGrade()
grader.displayGrader()

File: grader.js
const readine = required("readline")

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
})

rl.question("Enter student name: ",(name) =>{
  rl.question("Enter assignment name: ",(assignment)=>{
    rl.question("Enter grade in numbers: ",(grade)=>{

      rl.close();
    }
  })
})
  
class Grader{
  getGrade(grade){
    if(grade <= 100)&& (grade >= 90){
      grade = 'A'
    }
    elseif(grade <= 89)&& (grade >= 80){
      grade = 'B'
    }
    elseif(grade <= 79)&& (grade >= 70){
      grade = 'C'
    }
    elseif(grade <= 69)&& (grade >= 60){
      grade = 'D'
    }
    elseif(grade <60){
      grade = 'F'
    }
    else{
      "Please enter a valid number for grade"
    }
    return grade
  }
  displayGrader(){
  return "Grade Details: "+name+"@fullsail.edu"
"Your grade for "+assignment+" is: "+grade
}



#module.exports = Grader;
grader = new Grader();
grader.getGrade(grade);
console.log(grader.displayGrader());
          